cottoncandy/interfaces.py

```diff
@@ -410,7 +416,8 @@ def download_json(self, object_name):
         json_data : dict
             Dictionary representation of JSON file
         """
-        assert self.exists_object(object_name)
+        if not self.exists_object(object_name):
```

Comment:
Thought about this more. It'd be cleaner to add a function `assert_object_exists()` in `utils`, then just use that in all these places.